lib/EvalSourceMapDevToolModuleTemplatePlugin.js

```diff
@@ -55,7 +55,7 @@ class EvalSourceMapDevToolModuleTemplatePlugin {
 			sourceMap.sources = moduleFilenames;
 			if(sourceMap.sourcesContent) {
 				sourceMap.sourcesContent = sourceMap.sourcesContent.map(function(content, i) {
-					return `${content}\n\n\n${ModuleFilenameHelpers.createFooter(modules[i], this.requestShortener)}`;
+					return `{${content}\n}\n\n\n${ModuleFilenameHelpers.createFooter(modules[i], this.requestShortener)}`;
```

Comment:
this is the wrong location. I guess you wanted to add it in line 67.

but this breaks the source map...